.github/workflows/dockerhub-dev.yml

```diff
@@ -0,0 +1,21 @@
+name: Docker Image CI
+
+on:
+  push:
+    branches:
+      - dev
+
+jobs:
+  build:
+    if: github.repository == 'ScilifelabDataCentre/dds_web'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Publish to Docker Hub
+        uses: elgohr/Publish-Docker-Github-Action@master
+        with:
+          name: scilifelabdatacentre/dds-backend
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
```

Comment:
Just curious, this values are taken from ?

Reply:
They are defined in the org, added sharing to this repository as well.

Reply:
Great!